superset-frontend/cypress-base/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "run cypress against superset",
   "scripts": {
-    "cypress": "cypress",
+    "cypress": "CYPRESS_BASE_URL=\"http://localhost:${npm_config_port:-8088}\" cypress",
```

Comment:
This isn't necessary, but an extra addition to potentially minimize confusion for engineers that have an existing cypress test workflow and aren't running on port 8088.

Reply:
I think for `cypress open --port=8090`, the port is for Cypress itself, not the service it's testing against.

Reply:
Yes, correct, well, it's telling Cypress what url to test against, and would likely reconcile with the port that is exposed from however the flask app was started.